<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_panel_stack</class>
 <widget class="QDialog" name="config_panel_stack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>166</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cb_tra">
      <item>
       <property name="text">
        <string>UDP</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_net">
      <item>
       <property name="text">
        <string>SRT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DRT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VBF</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_mac">
      <item>
       <property name="text">
        <string>Aloha (without Ack)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UWAloha (with Ack)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XALoha (UEP)</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_phy">
      <item>
       <property name="text">
        <string>Simulation Channel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modem-QPSK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modem-MFSK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modem-MFSK_1710</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_compile">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TRA :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NET :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>MAC :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PHY :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Stack</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_current_phy">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_current_mac">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_current_net">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_current_tra">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config_panel_stack</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config_panel_stack</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
